guides/common/modules/proc_comparing-content-view-versions.adoc

```diff
@@ -4,7 +4,10 @@
 Use this procedure to compare Content View Version functionality for {Project}.
 
 .Procedure
-* In the {ProjectWebUI}, navigate to *Content* > *Content Views*, then select two Content View versions and click *Compare*.
+. In the {ProjectWebUI}, navigate to *Content* > *Content Views*.
+. Select the Content View versions you want to compare.
+. On the Versions tab, select the checkbox next to any two versions you want to compare.
```

Comment:
Actually, looking at it again, if these two steps specify the same thing, I would just keep the second one:
```suggestion
. On the *Versions* tab, select the checkbox next to any two Content View versions you want to compare.
```